vector/build.gradle

```diff
@@ -97,9 +97,17 @@ android {
         javaMaxHeapSize "2g"
     }
 
+    signingConfigs {
+        release {
+            storeFile file("act_riot_key_store.jks")
+            storePassword '123456'
```

Comment:
This isn't secure having passwords in free text in an open source project.  Is this able to be something more like https://stackoverflow.com/a/47231306 ?

Reply:
I think we will be using some other keys rather than using "act_riot_key_store.jks" and better password. Currently, I have done that way for appcenter build only, otherwise, you have to manually upload the key file to the app center and give the password.